FreeRTOS/FreeRTOS-Kernel
# Update the memory alignment within the Cortex-A9 port asm code

Description
-----------
Update the assembly code used in the Cortex-A9 port to ensure stack alignment before calling vTaskSwitchContext() in case user code added to the function (maybe via a trace macro or other mechanism) requires the more strict alignment.  That would be the case if, as an example, floating point instructions were used.

Test Steps
-----------
The self monitoring integration tests have been running for several weeks with no issues.



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
